gen_cert.py

```diff
@@ -166,18 +170,25 @@ def interstitial_factory():
         # If TEMPLATEFILE is set in cert-data.yml, this value has top priority.
         # Else if a value is passed in to the constructor (eg, from xqueue), it is used,
         # Else, the filename is calculated from the version and course_id.
+
         template_pdf = cert_data.get('TEMPLATEFILE', template_pdf)
-        template_prefix = '{0}/v{1}-cert-templates'.format(TEMPLATE_DIR, self.template_version)
-        template_pdf_filename = "{0}/certificate-template-{1}-{2}.pdf".format(template_prefix, self.org, self.course)
-        if template_pdf:
-            template_pdf_filename = "{0}/{1}".format(template_prefix, template_pdf)
-            if 'verified' in template_pdf:
-                self.template_type = 'verified'
-        try:
-            self.template_pdf = PdfFileReader(file(template_pdf_filename, "rb"))
-        except IOError as e:
-            log.critical("I/O error ({0}): {1} opening {2}".format(e.errno, e.strerror, template_pdf_filename))
-            raise
+
+        if template_pdf is not False:
+            template_prefix = '{0}/v{1}-cert-templates'.format(TEMPLATE_DIR, self.template_version)
```

Comment:
While we're editing this string template, let's use named kwargs instead of positional options.